client/src/routes/settings/+page.svelte

```diff
@@ -21,11 +54,165 @@
       errorMsg = err instanceof Error ? err.message : String(err);
     }
   }
+
+  function linkGoogleAccount() {
+    try {
+      linkWithGoogle();
+      googleLinked = true;
```

Comment:
This displays I successfully linked google account even though i only clicked the button.

Reply:
Fixed